spec/beaker/subcommand_spec.rb

```diff
@@ -32,6 +32,72 @@ module Beaker
       end
     end
 
+    context 'ensure that beaker options can be passed through' do
+
+      BEAKER_OPTIONS_LIST = [
```

Comment:
any reason this is a constant instead of a `let(:)`?

Reply:
I started this thinking this list would be useful for more than just this test, but I wound up deciding it should be limited to just this test for now. I"ll change it.